src/screens/CurrencySelectModal.js

```diff
@@ -23,7 +23,7 @@ import {
   CurrencySelectModalHeader,
   ExchangeSearch,
 } from '../components/exchange';
-import NetworkSwitcher from '../components/exchange/NetworkSwitcher';
+import { NetworkSwitcher as NetworkSwitcherv1 } from '../components/exchange/NetworkSwitcher';
```

Comment:
i don't think `../components/exchange/NetworkSwitcher` exists? the app is crashing

Reply:
ah no is not that, the app is crashing on my end when the crosschain swaps flag is off

Reply:
fixed